From b62c6a39a2c6efaa75010429e9dc5aacd13a39de Mon Sep 17 00:00:00 2001
From: Jerome Brunet <jbrunet@baylibre.com>
Date: Sat, 28 Mar 2020 21:56:04 +0100
Subject: [PATCH] meson: env: remove amlogic directory from fdtfile variable

The "amlogic" directory with fdtfile is messing with yocto extlinux
default configuration. It does not seems very useful and I'm to lazy
to look at extlinux right now

Signed-off-by: Jerome Brunet <jbrunet@baylibre.com>

---
 include/configs/meson64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index 52cc01f73da3..93e70fe2b72f 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -81,7 +81,7 @@
 	"kernel_addr_r=0x08080000\0" \
 	"pxefile_addr_r=0x01080000\0" \
 	"ramdisk_addr_r=0x13000000\0" \
-	"fdtfile=amlogic/" CONFIG_DEFAULT_DEVICE_TREE ".dtb\0" \
+	"fdtfile=" CONFIG_DEFAULT_DEVICE_TREE ".dtb\0" \
 	BOOTENV
 #endif
 
